OpenDreamShared/Json/DreamObjectJson.cs

```diff
@@ -20,6 +20,8 @@ public sealed class DreamTypeJson {
         public List<int> Verbs { get; set; }
         public Dictionary<string, object> Variables { get; set; }
         public Dictionary<string, int> GlobalVariables { get; set; }
+        public HashSet<string> ConstVariables { get; set; }
+        public HashSet<string> TmpVariables { get; set; }
```

Comment:
Since [the compiler doesn't always set these](https://github.com/OpenDreamProject/OpenDream/pull/1443/files#diff-76e845bf829013a0d2cf4684050050d5ccf1eb3e24b46200944b951b233f2ca3R174-R181) they can be null
```suggestion
        public HashSet<string>? ConstVariables { get; set; }
        public HashSet<string>? TmpVariables { get; set; }
```